update_prevalence.py

```diff
@@ -621,13 +621,29 @@ def main() -> None:
                 if (
                     place.population == 0
                     and place.name not in ("Unassigned", "Unknown")
+                    and place.name != "New York City"  # Hack: see below
                 ):
                     raise ValueError(
                         f"JHU data has cases but no population for {place!r}"
                     )
                 place.cumulative_cases[current] = line.Confirmed
             current -= timedelta(days=1)
 
+        # Hack: On 2020-09-01, JHU switched from reporting all NYC cases
+        # under a virtual county called "New York City" to reporting them
+        # under the counties for individual boroughs. Until 2020-09-14, we'll
+        # support mixing the pre- and post-transition data by merging the
+        # five borough counties (New York, Kings, Queens, Richmond, Bronx)
+        # into "New York City".
+        nyc_county_names = ("New York", "Kings", "Queens", "Richmond", "Bronx")
+        ny_state = data.countries["US"].states["New York"]
+        nyc_indv = [ny_state.counties.pop(cname) for cname in nyc_county_names]
+        nyc_combined = ny_state.counties["New York City"]
+        if not nyc_combined.population:
+            nyc_combined.population = sum(c.population for c in nyc_indv)
+        for county in nyc_indv:
```

Comment:
Use a list comprehension and sum() for consistency with above?

Reply:
Some of the cases are already filed under `nyc_combined`, so it's a different situation than for population. I guess we could write `nyc_combined.cumulative_cases += sum([c.cumulative_cases for c in nyc_indv], collections.Counter())` or `nyc_combined.cumulative_cases = sum([c.cumulative_cases for c in nyc_indv], nyc_combined.cumulative_cases)` but either of those seems harder to understand to me.

Reply:
Wow, that's confusing. Consider leaving a comment saying that some are populated.